### ISSUE TITLE
[BUG] Heading in the Learn tab not clearly visible in the light theme

### ISSUE THREAD

**Issue by Satyasn01** [codingkatty/mathpylearn#77]
![image](https://github.com/user-attachments/assets/d595bde2-ade0-40b7-9506-8cdf54b75272)

**Describe the bug:**  The heading in the "Learn" tab becomes unclear and hard to read when the light theme is active. The color palette used for the heading text doesn't provide enough contrast against the background, making it difficult to distinguish.

**How are you going to fix it?:**  I will update the CSS to use a more visible color for the heading text when the light theme is applied. This can be achieved by increasing the contrast between the text color and the background color, ensuring that the heading remains readable in both dark and light themes. For example, changing the text color to a darker shade like #333 or adjusting the background color for better contrast.

**Comment by codingkatty** [comment]
Do you think you can use the same navbar for all pages?

**Comment by Satyasn01** [comment]
@codingkatty you want this to be fixed?
if yes let me know your preference i will fix it

**Comment by codingkatty** [comment]
Yes, it's going to add the quality of the contribution, if you want you can do it, or open another issue (someone else could also do that issue, it's your choice to whether do it or not)
